h/services/search_index/_queue.py

```diff
@@ -22,11 +19,13 @@ class Priority:
 
     class Result:
         # Values for reporting which should stringify nicely
-        DELETED_FROM_DB = "Jobs deleted because annotations were deleted from the DB"
-        MISSING = "Annotations synced because they were not in Elasticsearch"
-        DIFFERENT = "Annotations synced because they were different in Elasticsearch"
-        UP_TO_DATE = "Jobs deleted because annotations were up to date in Elasticsearch"
-        FORCED = "Annotations synced because their jobs had force=True"
+        DELETED_FROM_DB = "jobs_deleted_because_annotations_were_deleted_from_DB"
+        MISSING = "annotations_indexed_because_they_were_missing_from_Elasticsearch"
+        DIFFERENT = "annotations_indexed_because_they_were_different_in_Elasticsearch"
+        UP_TO_DATE = "jobs_deleted_because_annotations_were_up_to_date_in_Elasticsearch"
+        FORCED = "annotations_indexed_and_jobs_deleted_because_they_had_force_True"
+        TOTAL_INDEXED = "annotations_indexed"
+        TOTAL_COMPLETED = "jobs_completed"
```

Comment:
I think these strings are trying to be keys and documentation at the same time, and don't make very nice metric names. They also don't group meaningfully.

I'd suggest having shorter names which say "<WhatHappened>/<Why>" which gets the message across and also makes for nice graphs in New Relic should we want it:

```suggestion
        # Annotations synced because they were not in Elasticsearch
        MISSING = "Synced/Missing"
        # Annotations synced because they were different in Elasticsearch
        DIFFERENT = "Synced/Different"
        # Annotations synced because their jobs had force=True
        FORCED = "Synced/Forced"
        TOTAL_INDEXED = "Synced/Total"

        # Jobs deleted because annotations were up to date in Elasticsearch
        UP_TO_DATE = "Completed/UpToDate"
        # Jobs deleted because annotations were deleted from the DB
        DELETED_FROM_DB = "Completed/DeletedFromDB"
        TOTAL_COMPLETED = "Completed/Total"
```

I'd also suggest changing `TOTAL_INDEXED` to `TOTAL_SYNCED`, although our terminology is a bit floppy in general around this.